extensions/interactions/GraphInput/GraphInput.py

```diff
@@ -107,6 +107,13 @@ class GraphInput(base.BaseInteraction):
             'type': 'bool',
         },
         'default_value': True
+    }, {
+        'name': 'interactionIsActive',
```

Comment:
Nope, don't do this. The change should be strictly a frontend one. The backend shouldn't be affected.

Reply:
Done